src/workerd/io/trace-stream.c++

```diff
@@ -993,11 +981,7 @@ kj::Promise<WorkerInterface::CustomEvent::Result> TailStreamCustomEventImpl::sen
   try {
     co_await sent.ignoreResult().exclusiveJoin(kj::mv(completionPaf.promise));
   } catch (...) {
-    auto e = kj::getCaughtExceptionAsKj();
-    if (revokePaf.fulfiller->isWaiting()) {
-      revokePaf.fulfiller->reject(kj::cp(e));
-    }
-    kj::throwFatalException(kj::mv(e));
+    kj::throwFatalException(kj::getCaughtExceptionAsKj());
```

Comment:
Just remove the try-catch statement. No point to catch only to rethrow

Reply:
I agree with Dan.